state-transition/core/state/statedb.go

```diff
@@ -268,16 +268,16 @@ func (s *StateDB[
 			return nil, err
 		}
 
-		withdrawalAddress, err = validator.
```

Comment:
We need to add `gocognit` to the list of no-lint for this function

Reply:
done